e3db/types/search.py

```diff
@@ -7,10 +7,44 @@ def __init__(self, last_index=0, count=50, include_all_writers=False, include_da
         """
         Initialize the search v2 class.
 
-        This object type is used to build a v2/search query to run against the server.
+        This object type is used to build a v2/search query for the search service.
```

Comment:
Same as other comment, remove Search Service in favor of either "server" or "E3DB"